Spigot-Server-Patches/0384-Anti-Xray.patch

```diff
@@ -28,6 +28,7 @@ index 48676152152faf7a7b9524ac37d8b4a8c32c4e2c..ca2ac17747d4f2acf1df056759c5c182
 +    public EngineMode engineMode;
```

Comment:
Use default values here and reference the variable as defaults when getting the values.

Reply:
There is no distinction, it is author preference. Given the rest of antixray config is written this way I see no reason he needs to change it now.

Reply:
Was under the impression that was the "standard" :p

Reply:
Not sure why this is necessary suddenly. Most other config options also don't do this. Also this can be done in another PR if needed.

Reply:
as said its not a standard, either way is fine. I personally do mine with the other pattern as sometimes what default you use in the method code might vary (for migrations).

Additionally, the method might be a little more complex and big, so I find it easier to view the defaults in the initializer of the variable.

but no reason to update pr over this, both is valid.